docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -8,6 +8,57 @@ sidebar_label: Time Series
 
 Below functions are used for time series analysis.
 
+## seriesOutliersTukey
+
+Detects a possible anomaly in series using [Tukey Fences](https://en.wikipedia.org/wiki/Outlier#Tukey%27s_fences).
+
+**Syntax**
+
+``` sql
+seriesOutliersTukey(series);
```

Comment:
Looking at [this](https://aakinshin.net/posts/tukey-outlier-probability/), it might make sense to give the user choice in specifying `k`. In other languages (e.g. [R](https://search.r-project.org/CRAN/refmans/lares/html/outlier_turkey.html)), `k` can be specified as an optional parameter.

But I don't mind too much. If you think that fixed `k = 1.5` is okay, that is also fine with me. In that case, please document the choice then here.